Swift/Tests/SampleCodeTest.swift

```diff
@@ -38,6 +38,18 @@ class SampleCodeTest: CBLTestCase {
         }
         // # end::new-database[]
     }
+    
+    #if COUCHBASE_ENTERPRISE
+    func dontTestDatabaseEncryption() throws {
+        database = self.db
+        
+        /* EE feature: code below might throw a compilation error
+         if it's compiled against CBL Swift Community. */
+        // # tag::database-encryption[]
+        // TODO
```

Comment:
@pasin could you fill in this example please? I'm not sure how to build CBL-EE Swift so i don't have auto completion for that feature.